service/authorization/v2/authorization.go

```diff
@@ -23,41 +27,97 @@ type Service struct {
 	config *Config
 	logger *logger.Logger
 	trace.Tracer
+	cache *EntitlementPolicyCache
+
+	// client managed by platform, provided by registration params
+	platformCacheClient *cache.Cache
+}
+
+type Config struct {
+	// Interval in seconds to refresh the in-memory policy entitlement cache (attributes and subject mappings)
+	// Default: cache is disabled with refresh interval set to 0.
+	CacheRefreshIntervalSeconds int `mapstructure:"cache_refresh_interval_seconds" default:"0"` // Cache disabled by default
```

Comment:
We should have a more sensible default, and separate out an `enabled/disabled` state so that customers only have to worry about "is caching happening" rather than "what is reasonable caching"

Reply:
We should also track work to test our CI pipeline against caching with eventual consistency in mind